applications/app/views/fragments/crosswords/crosswordMetaHeader.scala.html

```diff
@@ -1,3 +1,4 @@
+@import views.html.fragments.crosswords.crosswordMeta
```

Comment:
What's this import used for?

Reply:
It is used [here](https://github.com/guardian/frontend/pull/26924/files#diff-b135c51d479f9c15e4dc7c25a525af61bf149a61855e81103c43274ae01dfc31R28). I've noticed many times when a twirl template refers to another one the import is missing and IntelliJ suggests to import it

<img src="https://github.com/guardian/frontend/assets/19683595/473ed63b-9b43-4a95-ac73-d0df286c74af" alt="image" width="400" height="auto">

I don't know how it works without the import!